### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MuskanPaliwal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ An implementation of Minimax AI Algorithm on Tic-Tac-Toe (or Noughts and Crosses
 	<img src="preview/minimax_img.png"></img>
 </p>
 
-## What is Minimax?
-Minimax is a artificial intelligence applied in two player games, such as tic-tac-toe, checkers, chess and go. This games are known as zero-sum games, because in a mathematical representation: one player wins (+1) and other player loses (-1) or both of anyone not to win (0).
+## What is Minimax
```
Comment (avikalpg) [inline README.md:11]: Why did you remove the question mark from a statement which is obviously a question? Just because it is a title?

Comment (MuskanPaliwal) [reply README.md:11]: No no..I was actually using this repo for the hunk-webhook feature testing..I should have marked it as do not merge
